addons/web/static/src/views/calendar/calendar_model.js

```diff
@@ -504,10 +504,6 @@ export class CalendarModel extends Model {
             (fieldMapping.all_day && rawRecord[fieldMapping.all_day]) ||
             false;
 
-        if (isAllDay) {
-            start = start.startOf("day");
-            end = end.startOf("day");
-        }
```

Comment:
What does it change?

Reply:
We don't want the start to be set to midnight if  allday_display but not allday (so that the shown starting time is not midnight) That being said, I figured i should just define 'allday' above and use it instead